src/pages/statistics.tsx

```
@@ -1,431 +1,462 @@
+								<span className="pl-1">Community Insights</span>
+							</div>
+						</Link>
+						<SwitchPlatform />
+					</Stack>
+				</Stack>
+				<CustomTab
+					activeTab={activeTab}
+					onTabChange={handleTabChange}
+					labels={["Active members", "Disengaged members"]}
+					content={[
+						<div className="flex flex-col space-y-4">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)